skills/unraid/scripts/dashboard.sh

```
@@ -0,0 +1,214 @@
+    echo "- **CPU:** Model $CPU_MODEL ($CPU_CORES cores / $CPU_THREADS threads) - **${CPU_LOAD}% load**" >> "$OUTPUT_FILE"
+    echo "- **Memory:** ${TOTAL_MEM_GB}GB - **${MEM_USED_PCT}% used**" >> "$OUTPUT_FILE"
+    echo "" >> "$OUTPUT_FILE"
+
+    # Array capacity
+    ARRAY_TOTAL=$(echo "$RESPONSE" | jq -r '.data.array.capacity.kilobytes.total')
+    ARRAY_FREE=$(echo "$RESPONSE" | jq -r '.data.array.capacity.kilobytes.free')
+    ARRAY_USED=$(echo "$RESPONSE" | jq -r '.data.array.capacity.kilobytes.used')
+    
+    if [ "$ARRAY_TOTAL" != "null" ] && [ "$ARRAY_TOTAL" -gt 0 ]; then
+        ARRAY_TOTAL_GB=$((ARRAY_TOTAL / 1024 / 1024))
+        ARRAY_FREE_GB=$((ARRAY_FREE / 1024 / 1024))
```

Shellcheck (0.11.0)

[warning] ARRAY_FREE_GB appears unused. Verify use (or export if used externally).

(SC2034)